action/protocol/staking/handlers.go

```diff
@@ -205,7 +199,11 @@ func (p *Protocol) handleWithdrawStake(ctx context.Context, act *action.Withdraw
 		}
 	}
 
-	if blkCtx.BlockTimeStamp.Before(bucket.UnstakeStartTime.Add(p.config.WithdrawWaitingPeriod)) {
+	withdrawWaitTime := p.config.WithdrawWaitingPeriod
+	if !postFbkMigration {
+		withdrawWaitTime = _withdrawWaitingTime
+	}
+	if blkCtx.BlockTimeStamp.Before(bucket.UnstakeStartTime.Add(withdrawWaitTime)) {
```

Comment:
verified that p.config.WithdrawWaitingPeriod is only used here
in both master and r0.11